ts/IterableInboxMessageListItem.tsx

```diff
@@ -7,24 +7,53 @@ import {
    StyleSheet
 } from 'react-native'
 
-import Message from './messageType'
+import InboxRowViewModel from './InboxRowViewModel'
 import Customization from './customizationType'
 
 type MessageListItemProps = {
-   message: Message,
+   message: InboxRowViewModel,
    customization: Customization
 }
 
 const IterableInboxMessageListItem = ({ message, customization }: MessageListItemProps) => {
-   const messageTitle = message.inboxMetadata.title
-   const messageBody = message.inboxMetadata.subtitle
-   const messageCreatedAt = message.createdAt
+   const messageTitle = message.inAppMessage.inboxMetadata.title
+   const messageBody = message.inAppMessage.inboxMetadata.subtitle
+   const messageCreatedAt = convertTS(message.createdAt)
```

Comment:
Here, we should just pass along `new Date(message.createdAt)` rather than formatting it and doing all that, since the customer will specify how they will want that formatted.

Reply:
Ok I got rid of the formatting. The message list item just renders the timestamp as is. I believe we want to do this instead of creating a Date object. The Date object will be used on future formatting functionality.